src/concat.js

```diff
@@ -13,8 +13,8 @@ var invoker = require('./invoker');
  * @category List
  * @sig [a] -> [a] -> [a]
  * @sig String -> String -> String
- * @param {Array|String} a
- * @param {Array|String} b
+ * @param {Array|String|Object} a
+ * @param {Array|String|Object} b
```

Comment:
The problem with including `Object` here is we lose the ability to indicate that a dispatching function such as `R.map` also supports plain objects. I don't feel strongly about this, though, as with JSDoc there'll always be some ambiguity.